src/config/constants.ts

```diff
@@ -3,9 +3,8 @@ import chains from './chains'
 export const IS_PRODUCTION = !!process.env.NEXT_PUBLIC_IS_PRODUCTION
 export const IS_DEV = process.env.NODE_ENV === 'development'
 
-export const GATEWAY_URL_PRODUCTION =
-  process.env.NEXT_PUBLIC_GATEWAY_URL_PRODUCTION || 'https://safe-client.safe.global'
-export const GATEWAY_URL_STAGING = process.env.NEXT_PUBLIC_GATEWAY_URL_STAGING || 'https://safe-client.staging.5afe.dev'
+export const GATEWAY_URL_PRODUCTION = process.env.NEXT_PUBLIC_GATEWAY_URL_PRODUCTION || 'http://localhost:3030'
```

Comment:
Not review related: did you change the default gateway port? It opens on `3000` for me.

Reply:
Yes, I used a custom port 3030 with `APPLICATION_PORT=3030 yarn start:dev`